go.mod

```diff
@@ -73,4 +73,8 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.21.3
+replace (
+	github.com/open-cluster-management/api => open-cluster-management.io/api v0.0.0-20210513122330-d76f10481f05
+	github.com/open-cluster-management/klusterlet-addon-controller => github.com/stolostron/klusterlet-addon-controller v0.0.0-20210303215539-1d12cebe6f19
```

Comment:
Instead of replace can we just use `open-cluster-management.io/api` and `github.com/stolostron/klusterlet-addon-controller`

Because in the pkg level import code section we are already importing `stolostron` like
```
import v1 "github.com/stolostron/multicloud-operators-placementrule
```

So it seems like we are a little bit inconsistent. I am good if we keep it like this if you think it's less edits in the package level code which makes it a safer change. Thanks.

Reply:
As discussed, its safer to use replace.

/approve

/lgtm